app/controllers/application_controller.rb

```diff
@@ -348,6 +348,8 @@ def self.requires_plugin(plugin_name)
     before_action do
       if plugin = Discourse.plugins_by_name[plugin_name]
         raise PluginDisabled.new if !plugin.enabled?
+      elsif Rails.env.test?
+        raise "Required plugin '#{plugin_name}' not found"
```

Comment:
Maybe add a little more info to help people fix the problem?
```suggestion
        raise "Required plugin '#{plugin_name}' not found. The string passed to requires_plugin should match the plugin's name at the top of plugin.rb"
```